Docs: Cleanup selinux_permissive module docs

major opened the pull request:
**SUMMARY**

This is a small documentation cleanup for the `selinux_permissive` module.

**ISSUE TYPE**
- Docs Pull Request

**COMPONENT NAME**
selinux_permissive

**ANSIBLE VERSION**
```paste below
ansible 2.6.3
  config file = None
  configured module search path = [u'/home/major/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/major/.pyenv/versions/2.7.14/envs/venv-2.7.14/lib/python2.7/site-packages/ansible
  executable location = /home/major/.pyenv/versions/venv-2.7.14/bin/ansible
  python version = 2.7.14 (default, Mar  7 2018, 08:13:01) [GCC 4.2.1 Compatible Clang 5.0.1 (tags/RELEASE_501/final)]
```

mscherer commented:
lgtm

acozine commented on lib/ansible/modules/system/selinux_permissive.py:
```diff
@@ -19,35 +19,36 @@
-        - "automatically reload the policy after a change"
-        - "default is set to 'false' as that's what most people would want after changing one domain"
-        - "Note that this doesn't work on older version of the library (example EL 6), the module will silently ignore it in this case"
+        - Automatically reload the policy after a change.
```
@major I find this confusing, and I'm hoping you can help clarify the behavior. The parameter is called `no_reload` (instead of `reload`), and the double negative seems like it should behave thus:
- if `no_reload` is set to `no`, the module reloads
- if `no_reload` is set to `yes`, the module does not reload
But the line below this suggests that when `no_reload` is set to `no`, the module does NOT reload. I know you're not changing the parameter in this PR, but can you clarify the behavior?

acozine commented on lib/ansible/modules/system/selinux_permissive.py:
```diff
@@ -19,35 +19,36 @@
 '''
 
 EXAMPLES = '''
+# Change the 'httpd_t' domain to permissive
```
Could you convert this comment into a `name:` line?

acozine commented:
I should have started by saying, thank you @major for updating the documentation! We appreciate your attention to detail.

major commented:
@acozine Good points there. I'll go back and rework the patch.

major committed:
Docs: Cleanup selinux_permissive module docs

Update the selinux_permissive instructions and add a comment to the
provided example.

Signed-off-by: Major Hayden <major@redhat.com>

acozine commented:
Thanks @major
